boskos/janitor/janitor.go

```diff
@@ -61,11 +68,11 @@ func setup(c boskosClient, janitorCount int, bufferSize int) chan string {
 	return buffer
 }
 
-func run(c boskosClient, buffer chan string) int {
+func run(c boskosClient, buffer chan string, res string) int {
 	totalAcquire := 0
 
 	for {
-		if proj, err := c.Acquire("project", "dirty", "cleaning"); err != nil {
+		if proj, err := c.Acquire(res, "dirty", "cleaning"); err != nil {
```

Comment:
Given that what type of resource you get appears to be the only difference it seems like you could make the same janitor handle cleaning both gke and gce projects (say by making --resource-type split by "," or append to a list rather than set a string [see kubetest for examples of appending]).

What made you decide to go with two deployments, one per type?